hw/dv/tools/vcs/unr.cfg

```diff
@@ -0,0 +1,23 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+-covInput ../../../cov_merge/merged.vdb
+-covDUT tb.dut
+
+# Provide the clock specification
+-clock clk 100
+# Provide the reset specification: signal_name, active_value, num clk cycles reset to be active
+-reset rst_n 0 20
+
+# Black box some of the modules
+# -blackBoxes -type design *
+
+# Include common el file, so that it doesn't generate reviewed common exclusions
+-covEL ../../vcs/common_cov_excl.el
```

Comment:
Please add TODO to figure out if there is a better way than using relative paths.

Reply:
use env var to avoid this. Updated.